src/mscorlib/Tools/BclRewriter/BclRewriter.targets

```diff
@@ -5,26 +5,33 @@
   <!-- ********************************************************************************************* -->
 
   <PropertyGroup>
-    <BclRewriterCommand Condition="'$(BclRewriterCommand)'==''">$(CoreCLRToolsDir)\BclRewriter.exe</BclRewriterCommand>
+    <BaseOutputPath Condition="'$(BaseOutputPath)' == ''">$(__BinDir)</BaseOutputPath>
     <BclRewriterModelFile>$(MscorlibDir)model.xml</BclRewriterModelFile>
     <BclRewriterWorkDir>$(IntermediateOutputPath)\BclRewriter</BclRewriterWorkDir>
     <BclRewriterSymbolOutput>$(IntermediateOutputPath)\BclRewriter\$(TargetName).pdb</BclRewriterSymbolOutput>
     <BclRewriterOutput>$(IntermediateOutputPath)\BclRewriter\$(TargetName)$(TargetExt)</BclRewriterOutput>
+    <BuildToolsSemaphore>$(IntermediateOutputPath)\BclRewriter\semaphore</BuildToolsSemaphore>
   </PropertyGroup>
 
+  <Import Project="$(ToolsDir)toolruntime.targets" />
+  
   <ItemGroup>
     <RewrittenAssembly Include="$(BclRewriterOutput)" />
   </ItemGroup>
   
+  <PropertyGroup>
+    <BclRewriterCommand Condition="'$(BclRewriterCommand)'==''">"$(ToolRuntimePath)$(ToolHost)" "$(ToolsDir)BclRewriter.exe"</BclRewriterCommand>
```

Comment:
you are right I forgot to change that, although do you want me to do it on build.sh or should I do it in this file by checking if the os is ubuntu and overwriting the variable?